db/migrate/20211009182016_create_users.rb

```diff
@@ -1,15 +1,15 @@
-# frozen_string_literal: true
-
-class CreateUsers < ActiveRecord::Migration[6.1]
-  def change
-    create_table :users do |t|
-      t.string :username
-      t.string :firstname
-      t.string :lastname
-      t.string :bio, default: 'Please enter a bio'
-      t.integer :user_type, default: 1
-
-      t.timestamps
-    end
-  end
-end
+# frozen_string_literal: true
+
+class CreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+      t.string :username
+      t.string :firstname
+      t.string :lastname
+      t.string :bio, default: 'Please enter a bio'
+      t.integer :user_type, default: 1
+      t.integer :evisible, default: 1

```

Comment:
```suggestion
```
Make a new migration file for this, https://stackoverflow.com/questions/4834809/adding-a-column-to-an-existing-table-in-a-rails-migration